src/renderer/routes/flow_chart/views/env-var/EnvVarCredentialsInfo.tsx

```diff
@@ -24,27 +23,19 @@ const EnvVarCredentialsInfo = ({
   setSelectedCredential,
   setDeleteModalOpen,
   setEditModalOpen,
-}: EnvVarCredentialsInfoProps) => {
+}: Props) => {
   const [showPassword, setShowPassword] = useState<boolean>(false);
   const [credentialValue, setCredentialValue] = useState<string>(
     credential.value,
   );
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const { withPermissionCheck, isAdmin } = useWithPermission();
 
-  const toggleShowPassword = () => {
+  const toggleShowPassword = async () => {
     if (credential.value === "") {
       setIsLoading(true);
-      baseClient
-        .get(`env/${credential.key}`)
-        .then((res) => {
-          setCredentialValue(res.data.value);
-          setIsLoading(false);
-        })
-        .catch((err) => {
-          toast.error(err.response?.data?.detail ?? err.message);
-          setIsLoading(false);
-        });
+      const res = (await captain.get(`env/${credential.key}`).json()) as EnvVar;
+      setCredentialValue(res.value);
```

Comment:
Should handle error as well.

```suggestion
      setCredentialValue(res.value);
      setIsLoading(false)
```